specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/networkVerifier.json

```diff
@@ -0,0 +1,1247 @@
+            "name": "workspaceName",
+            "description": "Workspace name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          },
+          {
+            "in": "path",
+            "name": "reachabilityAnalysisRunName",
+            "description": "Reachability Analysis Run name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Analysis Run resource object to create/update.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReachabilityAnalysisRun"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ReachabilityAnalysisRun updated.",
+            "schema": {
+              "$ref": "#/definitions/ReachabilityAnalysisRun"
+            }
+          },
+          "201": {
+            "description": "ReachabilityAnalysisRun created and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ReachabilityAnalysisRun"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CommonErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "ReachabilityAnalysisRunCreate": {
+            "$ref": "./examples/ReachabilityAnalysisRunPut.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ReachabilityAnalysisRuns"
+        ],
+        "summary": "Deletes Reachability Analysis Run.",
+        "operationId": "ReachabilityAnalysisRuns_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "workspaceName",
+            "description": "Workspace name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          },
+          {
+            "in": "path",
+            "name": "reachabilityAnalysisRunName",
+            "description": "Reachability Analysis Run name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=11&length=3)

missing the

Reply:
added back, we initially were not going to support location headers for analysisrun delete and verifierworkspace delete but we are only not supporting location headers in analysis run put anymore